api/src/main/resources/liquibase.xml

```diff
@@ -322,86 +321,6 @@
 		</sql>
 	</changeSet>
 
-
-</databaseChangeLog>
```

Comment:
@Mwanje you have removed the public holiday change sets, you just need to remove a single tag